http/transport/default_transport.go

```diff
@@ -7,6 +7,22 @@ package transport
 // If not explicitly finalized via `Final` it uses `http.DefaultTransport` as finalizer.
 func NewDefaultTransportChain() *RoundTripperChain {
 	return Chain(
+		&ExternalDependencyRoundTripper{},
+		NewDefaultRetryRoundTripper(),
+		&JaegerRoundTripper{},
+		NewDumpRoundTripperEnv(),
+		&LoggingRoundTripper{},
+		&LocaleRoundTripper{},
+		&RequestIDRoundTripper{},
```

Comment:
Would it make sense to move the request id roundtripper up so it gets used by the other trippers?

Reply:
Maybe, didn't investigate that, so maybe open an issue for that.

Reply:
https://github.com/pace/bricks/issues/242